react/components/Payment/PaymentMethod.tsx

```diff
@@ -1,19 +1,16 @@
-import React, { Component } from 'react'
+import React, { FunctionComponent, useState } from 'react'
 import { InjectedIntlProps, injectIntl } from 'react-intl'
-import { IconCaretDown, IconCaretUp } from 'vtex.styleguide'
 
+import InfoIcon from '../../Icons/Info'
 import ButtonLink from '../ButtonLink'
+import AdditionalInfo from './AddittionalInfo'
 import Price from './FormattedPrice'
 
 interface Props {
   payment: Payment
   transactionId: string
 }
 
-interface State {
-  open: boolean
-}
-
 const paymentGroupSwitch = (payment: Payment, intl: any) => {
```

Comment:
`intl` is an object if i'm not mistaken, with functions like `formatMessage()` & `formatDate()`

Reply:
Ooops, this one slipped from me hahahah
`intl` in this case has type `ReactIntl.InjectedIntl` that comes from `@types/react-intl`